packages/components/src/range-control/index.native.js

```diff
@@ -55,6 +55,7 @@ function RangeControl( {
 			afterIcon={ afterIcon }
 			allowReset={ allowReset }
 			defaultValue={ initialSliderValue }
+			separatorType={ separatorType }
```

Comment:
Hopefully, this will be not needed when this PR https://github.com/WordPress/gutenberg/pull/21365 is merged.